README.md

```diff
@@ -105,7 +105,7 @@ jsonPayload.verb="stacktraceanalyzer"
 
 Here is the related PyPI package: https://pypi.org/project/cloud-tpu-diagnostics.
 
-## Ahead of Time Compilation (AOT, tpu-only)
+## Ahead of Time Compilation (AOT)
```

Comment:
How about refactoring the documentation for AOT in the following way?

1. Add a sub section `TPU Support` and move all the TPU related instructions and existing examples for AOT under this section.
2. For the newly added section `GPU Support`, add examples for GPUs (similar to the ones for TPUs, but with different flags).